src/test/java/seedu/spendnsplit/testutil/UpdatePortionDescriptorBuilder.java

```diff
@@ -41,7 +41,11 @@ public UpdatePortionDescriptorBuilder withPersonName(String name) {
      * Sets the {@code Weight} of the {@code UpdatePortionDescriptor} that we are building.
      */
     public UpdatePortionDescriptorBuilder withWeight(String weight) {
-        descriptor.setWeight(new Weight(weight));
+        try {
+            descriptor.setWeight(new Weight(weight));
+        } catch (Exception e) {
+            //
+        }
```

Comment:
Maybe we could use the `ignored` keyword for this case.

```java
        } catch (Exception ignored) {
            //
        }
```

Additionally, it would be nice to write a comment as to why this Exception is ignored.